pkg/app/app.go

```diff
@@ -18,7 +18,7 @@ type Config struct {
 	AppName             string
 	AppNamespace        string
 	AppVersion          string
-	ConfigVersion       string
+	ConfigMajorVersion  int
```

Comment:
Why that change? That will require changes in the architect binary as well. I'd rather drop this eventually. We probably aren't gonna use that.

Reply:
PTAL at revert commit: https://github.com/giantswarm/app/pull/62/commits/af2302e693c271cd91e0074407ddf50f1c2d9c34

Should I change it back to ConfigVersion string and keep the rest of the revert intact?

Reply:
I don't remember how it was included in a single PR, but it should be `ConfigVersion string`.